modules/Db2/scripts/install_Db2.sh

```diff
@@ -14,11 +14,18 @@
 CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 OC_CMD=oc
 
-echo
+. ../../modules/Db2/scripts/common-ocp-utils.sh
+
+
 echo "Creating Storage Class ..."
 ${OC_CMD} apply -f ${DB2_STORAGE_CLASS_FILE}
 sleep 10
 
+echo "Docker username: ${DOCKER_USERNAME}"
+${OC_CMD} create secret docker-registry ibm-registry --docker-server=${DOCKER_SERVER} --docker-username=${DOCKER_USERNAME} --docker-password=${ENTITLED_REGISTRY_KEY} --docker-email=${ENTITLEMENT_REGISTRY_USER_EMAIL} --namespace=ibm-db2
```

Comment:
You have hardcoded `ibm-db2` here and are referencing it before namespace is created on line 46.

Reply:
Fixed. I replaced them with `${DB2_PROJECT_NAME}`  and got it created before it's used.